src/ResourceManagement/ContainerRegistry/ContainerRegistry.Tests/Helpers/ContainerRegistryTestUtilities.cs

```diff
@@ -0,0 +1,145 @@
+﻿//
+// Copyright (c) Microsoft.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+using System.Collections.Generic;
+using Microsoft.Azure.Management.Resources;
+using Microsoft.Azure.Management.Resources.Models;
+using Microsoft.Azure.Management.Storage;
+using Microsoft.Azure.Management.Storage.Models;
+using Microsoft.Azure.Management.ContainerRegistry;
+using Microsoft.Azure.Management.ContainerRegistry.Models;
+using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using Xunit;
+
+namespace ContainerRegistry.Tests
+{
+    public class ContainerRegistryTestUtilities
+    {
+        public static string DefaultLocation = "southcentralus";
```

Comment:
Please do not encode a static default location.  It is best to query ARM for an appropriate location during test setup.  Using a static location causes maintenance problems when tests are run later - many environments do not contain this location, checking ARM for locations containing the needed RPs *always* works in every environment.